server/sonar-server/src/main/java/org/sonar/server/organization/OrganizationCreationImpl.java

```diff
@@ -220,6 +239,21 @@ private void insertGroupPermission(DbSession dbSession, PermissionTemplateDto te
     dbClient.permissionTemplateDao().insertGroupPermission(dbSession, template.getId(), group == null ? null : group.getId(), permission);
   }
 
+  private void insertQualityProfiles(DbSession dbSession, OrganizationDto organization) {
+    List<ActiveRuleChange> changes = new ArrayList<>();
+    definedQProfileRepository.getQProfilesByLanguage().entrySet()
+      .stream()
+      .flatMap(entry -> entry.getValue().stream())
+      .forEach(profile -> insertQualityProfile(dbSession, profile, organization, changes));
+    activeRuleIndexer.index(changes);
```

Comment:
Responsibility to index profile should be part of `DefinedQProfileCreation`. Moreover it will allow to refactor later the way to index Quality profiles (it's not necessarily based on list of changes).